sdk/core/azure-core-test/inc/azure/core/test/test_proxy_manager.hpp

```diff
@@ -56,7 +56,7 @@ namespace Azure { namespace Core { namespace Test {
     // Using a reference because the context lives in the test_base class and we don't want to make
     // a copy.
     Azure::Core::Test::TestContextManager& m_testContext;
-    const std::string m_proxy = "https://localhost:5001";
+    const std::string m_proxy = "http://localhost:5000";
```

Comment:
yes , macos latest version test-proxy doesn't work properly with the test proxy https endpoint. since this is internal local call i'm not worried abouyt it

Reply:
Why doesn't it work now? Can you share more details or the error you see with using https compared to when it worked previously?

Reply:
previously it was working as expected, now the error i see is cannot connect to https://localhost:5001 from curl , no more info as to why. i switched to http as it's what the pipelines use for the start / is alive / logs steps for the test-proxy when they start it. 
The error only happens on macOS 14 previously it was working, and i do not posses a mac dev box to debug the issue,